velox/exec/TableScan.cpp

```diff
@@ -179,7 +179,13 @@ RowVectorPtr TableScan::getOutput() {
         dataSource_->setFromDataSource(std::move(preparedDataSource));
       } else {
         curStatus_ = "getOutput: adding split";
+        const auto addSplitStartMicros = getCurrentTimeMicro();
```

Comment:
Nit: why we don't use MicrosecondTimer? Thanks!

Reply:
Use `MicrosecondTimer` here will just make the code longer with artificial scope and extra variable declaration